app/routes.js

```diff
@@ -167,6 +182,7 @@ export const getRoutes = (appContext) => {
     <Route path='/' component={AppComponent} {...props}>
       <IndexRoute components={{login:Login}} onEnter={onIndexRouteEnter(api)} />
       <Route path='login' components={{login:Login}} onEnter={requireNoAuth(api)} />
+      <Route path='terms' components={{terms:Terms}} />
```

Comment:
This struck me as odd at first, /terms being our only route w/o an `onEnter`, but after thinking it through, I guess it really is.